From 2aad0191c0b82fd6271be286ea48c131c64b3f21 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Tue, 25 Dec 2012 15:05:30 +0800
Subject: [PATCH 120/281] ENGR00238210-2 imx6 sabresd: set the default density
 to 140dpi

When set display density to 160dpi, the launcher's workspace
page will exceed the screen boarder, and caused the part
of some widget invisiable. Also on the status bar, the batter
icon bottom will be cut.
After tring some density, 140dpi is better than 160dpi and 128dpi.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/sabresd/init.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index 82447e8..f49ff2c 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -62,8 +62,8 @@ on boot
     # Set rotation to 270 to cofigure as portrait mode
     setprop ro.sf.hwrotation 0
 
-    # Set the density to 160dpi, default 128dpi is not good
-    setprop ro.sf.lcd_density 160
+    # Set the density to 140dpi, default 128dpi is not good
+    setprop ro.sf.lcd_density 140
 
 # change for FSL specific service
     chown system system /sys/devices/system/cpu/cpu0
-- 
1.8.0

